pyproject.toml

```diff
@@ -133,6 +137,7 @@ ignore = [
 # S101 - assert
 # B011 - assert-false
 "tests/*" = ["ANN001", "ANN2", "ANN102", "S101", "B011"]
+"src/ga4gh/cat_vrs/*models.py" = ["ANN102"]
```

Comment:
intentional? is this a submodules thing?

Reply:
Ope thank you. Copy/paste error